fastestimator/network.py

```diff
@@ -52,12 +55,23 @@ def _verify_inputs(self) -> None:
         Raises:
             AssertionError: If any of the ops are not TensorOps.
         """
-        for op in self.ops:
-            if isinstance(op, Scheduler):
-                for epoch_op in op.get_all_values():
-                    assert isinstance(epoch_op, TensorOp), "unsupported op format, must provide TensorOp in Network"
-            else:
-                assert isinstance(op, TensorOp), "unsupported op format, must provide TensorOp in Network"
+        for op in get_current_items(self.ops):
+            assert isinstance(op, TensorOp), "unsupported op format, must provide TensorOp in Network"
+
+    def get_schedule_items(self, mode: [str]) -> List[Any]:
```

Comment:
what is the typing on mode supposed to be? List[str] ??

Reply:
fixed, it should be str